.github/workflows/ci-tk.yaml

```diff
@@ -50,6 +50,11 @@ jobs:
           echo VIRTUAL_ENV=$VIRTUAL_ENV >> "$GITHUB_ENV"
           echo "$VENV_DIR/bin" >> "$GITHUB_PATH"
 
+      - name: "Setting up Rust"
+        uses: actions-rust-lang/setup-rust-toolchain@v1
```

Comment:
Please, add rust install instructions to Wave readme

Reply:
sure will do.